src/app/test_executive/test_executive.ml

```diff
@@ -437,28 +505,72 @@ let debug_arg =
 
 let help_term = Term.(ret @@ const (`Help (`Plain, None)))
 
+let info engine_name =
+  let doc =
+    match engine_name with
+    | "abstract" ->
+        "Run mina integration tests with the abstract engine."
+    | "cloud" ->
+        "Run mina integration tests on a remote cloud provider."
+    | _ ->
+        assert false
+  in
+  Term.info engine_name ~doc ~exits:Term.default_exits
+
 let engine_cmd ((engine_name, (module Engine)) : engine) =
-  let info =
-    let doc = "Run mina integration test(s) on remote cloud provider." in
-    Term.info engine_name ~doc ~exits:Term.default_exits
+  let info = info engine_name in
+  let module Inputs = Make_test_inputs (Engine) () in
+  let set_config path =
+    Option.iter path ~f:(fun p -> Engine.Network.config_path := p) ;
+    path
+  in
+  let set_keypair path =
+    Engine.Network.keypairs_path := path ;
+    path
+  in
+  let set_mina_image image =
+    Engine.Network.mina_image := image ;
```

Comment:
Why do we have this now? Previously GCP was using the mina_image and arcive_image, so what's changed? Thanks.

Reply:
Because we need this info inside the topology file which is produced in a module that `test_executive` depends on